src/microbe_stage/components/Engulfer.cs

```diff
@@ -189,6 +189,10 @@ public static EngulfCheckResult CanEngulfObject(this ref Engulfer engulfer, uint
             if (!compounds.Compounds.Any(p => usefulCompoundSource.IsUseful(p.Key)))
                 continue;
 
+            // Skip entities that are too small to catch easily
+            if (engulfer.EngulfingSize / .4 > engulfable.AdjustedEngulfSize)
```

Comment:
Putting this here will impact AI behaviour as well. Perhaps it would be good to have a new parameter for this method to "skipLikelyTooFastTargets`? That way the info that is fed through the tutorial to the player could use that flag and the microbe AI, for now at least, could keep the old behaviour?